cmd/internal/cli/build.go

```diff
@@ -40,8 +40,37 @@ var (
 	dockerPassword string
 	dockerLogin    bool
 	noCleanUp      bool
+
+	// AllowUnauthenticatedBuild When true, will never ask if you want to build
```

Comment:
go documentation strings should be written as complete sentences as in `AllowUnauthenticatedBuild determines whether ...`. Also, I don't think I understand the sentence "this only works when your bootstrap is the library".

And this variable should not be exported.

Reply:
This PR will likely be closed soon (no need to review anymore), and rewritten in a new cleaner PR (with less commits). Thanks for the suggestion!

Reply:
The "this only works when your bootstrap is the library" means that this feature will only work if you are building from the library, as opposed to docker, shub, etc...